packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/done.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { arrayOf, bool, func, number, shape, string } from 'prop-types';
+import { Edit2 as EditIcon } from 'react-feather';
+
+import { Price } from '@magento/peregrine';
+
+import { mergeClasses } from '../../../classify';
+import Icon from '../../Icon';
+import defaultClasses from './done.css';
+
+const Done = props => {
```

Comment:
This is a bit vague for a component name. Maybe iterate on it a bit?

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/2280/commits/db303d8d61e68255ebbb4a8d07a9f3f5eab599fe.